server_installation/topics/network/outgoing.adoc

```diff
@@ -74,6 +74,13 @@ To determine the proxy for an outgoing HTTP request the target hostname is match
 hostname patterns. The first matching pattern determines the proxy-uri to use.
 If none of the configured patterns match for the given hostname then no proxy is used.
 
+If the proxy server requires authentication, include the proxy user's credentials like `username:password@`, e.g.:
```

Comment:
Instead of "like" please use "in this format"
Also, always replace "e.g."  with For example.  And please make it a new sentence.

Reply:
OK. I understand.